doc/1/controllers/bulk/write-async/snippets/write-async.cs

```diff
@@ -0,0 +1,6 @@
+try {
+    await kuzzle.Bulk.WriteAsync("nyc-open-data", "yellow-taxi", JObject.Parse("{foo: 'bar'}"));
```

Comment:
An example with a `_kuzzle_info` property in the body is more appropriate since this is one of this route purpose